### ISSUE TITLE
Consider adding aliases (sorry to be that guy)

### ISSUE THREAD

**Issue by HenrikJoreteg** [lukekarrys/local-links#5]
Sorry to be that guy, feel free to tell me to bug off. 

I think this code:

``` js
handleClick: function (e) {
        var pathname = links.pathname(e);

        if (pathname) {
            e.preventDefault();
            app.navigate(pathname);
        }
    }
```

would be a lot more readable if it was:

``` js
handleClick: function (e) {
        var pathname = links.getLocalPathname(e);

        if (pathname) {
            e.preventDefault();
            app.navigate(pathname);
        }
    }
```

The first looks like it will just get the pathname from anything, rather than only returning a pathname if it's local.

I'd suggest re-naming or aliasing these two:

`pathname` => `getLocalPathname`
`active` => `isActive` (to match `isLocal`)

If i name the required module `localLinks` it's better `localLinks.pathname()` makes more sense. But I think it'd be clearer with the alias.

**Comment by HenrikJoreteg** [comment]
btw, i've been using this and liking it a lot.

**Comment by lukekarrys** [comment]
@HenrikJoreteg I don't know if you're using the `hash` function, but do you think that should be aliased too then? Maybe `getLocalHash()`?

**Comment by HenrikJoreteg** [comment]
Hmm... not entirely sure, actually. Maybe???

**Comment by lukekarrys** [comment]
I do like `getLocalHash` and `getLocalPathname` so I updated the docs to reflect those, and added the alias names to the docs as well.

Published as `1.3.0`.
